app/config/ApplicationConfig.scala

```diff
@@ -67,7 +68,8 @@ class ApplicationConfigImpl extends ApplicationConfig with ServicesConfig {
 
   def routeToSwitchLanguage = (lang: String) => routes.LanguageSwitchController.switchToLanguage(lang)
 
-  private lazy val _reportAProblemPartialUrl = s"$contactHost/contact/problem_reports?secure=false"
+  private lazy val _reportAProblemPartialUrl = s"$contactHost/contact/problem_reports_ajax?secure=false"
```

Comment:
This looks different than usual, why does it say secure=false? It's normally the service name or identifier like 'ers-checking', e.g.
```suggestion
lazy val reportAProblemPartialUrl: String = s"$contactHost/contact/problem_reports_ajax?service=$contactFormServiceIdentifier"
```

Reply:
Also not sure why there is an underscore before the name? Can we remove it?

Reply:
didn't change it as that's what it was before - just running tests to see if all's fine after refactoring.

Reply:
changed & pushed